scratch/paste-rename-poc/poc.css

```
@@ -0,0 +1,301 @@
+	--bg: #1e1e1e;
+	--bg-2: #252525;
+	--bg-3: #2b2b2b;
+	--border: #3f3f3f;
+	--text: #dadada;
+	--muted: #9b9b9b;
+	--faint: #6b6b6b;
+	--accent: #7f6df2;
+	--accent-hover: #8b7cf5;
+	--danger: #e46c6c;
+	--shadow: 0 12px 40px rgb(0 0 0 / 0.45);
+	font-family: "Inter", "Segoe UI", system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)